src/main/java/com/idrsolutions/microservice/utils/LibreOfficeHelper.java

```diff
@@ -61,7 +73,7 @@ public static boolean convertToPDF(final String sofficePath, final File file, fi
 
         try {
             final Process process = pb.start();
-            if (!process.waitFor(1, TimeUnit.MINUTES)) {
+            if (!process.waitFor(timeoutDuration, TimeUnit.MILLISECONDS)) {
                 process.destroy();
                 DBHandler.getInstance().setError(uuid, 1050, "Libreoffice timed out after 1 minute");
```

Comment:
@KieranFrance this error message needs to account for when the timeout has been configured to use a non-default value

Reply:
Good catch, updating it now